Google.Ads.GoogleAds/examples/CampaignManagement/ValidateAd.cs

```diff
@@ -164,13 +166,11 @@ public void Run(GoogleAdsClient client, long customerId, long adGroupId)
                             }
                             count++;
```

Comment:
I think you need to move this inside of the nested for loop. Right now, your count will always equal 1. To that end I would suggest reworking the console out slightly to

```
                                    Console.WriteLine($"{count}) Policy topic entries with topic = " +
                                        $"\"{entry.Topic}\" and type = \"{entry.Type}\" " +
                                        $"found.");
```

Reply:
Thanks, addressed this in #442.